src/patternfly/components/Masthead/masthead.scss

```diff
@@ -0,0 +1,200 @@
+
+  // insets
+  --pf-c-masthead--m-page-insets--inset: var(--pf-global--spacer--md);
+  --pf-c-masthead--m-page-insets--xl--inset: var(--pf-global--spacer--lg);
+
+  // stack
+  --pf-c-masthead--m-display-stack__main--Order: -1;
+  --pf-c-masthead--m-display-stack__main--FlexBasis: 100%;
+  --pf-c-masthead--m-display-stack__main--BorderBottom: var(--pf-c-masthead__main--BorderBottomWidth) solid var(--pf-c-masthead__main--BorderBottomColor);
+  --pf-c-masthead--m-display-stack__main--PaddingRight: var(--pf-c-masthead--inset);
+  --pf-c-masthead--m-display-stack__main--PaddingLeft: var(--pf-c-masthead--inset);
+  --pf-c-masthead--m-display-stack__content--PaddingRight: var(--pf-c-masthead--inset);
+  --pf-c-masthead--m-display-stack__content--PaddingLeft: var(--pf-c-masthead--inset);
+
+  // inline
+  --pf-c-masthead--m-display-inline__main--Order: 0;
+  --pf-c-masthead--m-display-inline__main--FlexBasis: auto;
+  --pf-c-masthead--m-display-inline__main--BorderBottom: 0;
+  --pf-c-masthead--m-display-inline__main--PaddingRight: var(--pf-global--spacer--md);
+  --pf-c-masthead--m-display-inline__main--PaddingLeft: var(--pf-global--spacer--md);
+  --pf-c-masthead--m-display-inline__content--PaddingRight: var(--pf-global--spacer--md);
+  --pf-c-masthead--m-display-inline__content--PaddingLeft: var(--pf-global--spacer--md);
+
+  // light 100
+  --pf-c-masthead--m-light-100--BackgroundColor: var(--pf-global--BackgroundColor--100);
+  --pf-c-masthead--m-light-100__main--BorderBottomColor: var(--pf-global--palette--black-200);
+
+  // light 200
+  --pf-c-masthead--m-light-200--BackgroundColor: var(--pf-global--BackgroundColor--200);
+  --pf-c-masthead--m-light-200__main--BorderBottomColor: var(--pf-global--palette--black-300);
+
+  @media screen and (min-width: $pf-global--breakpoint--xl) {
+    --pf-c-toolbar--m-page-insets--inset: var(--pf-c-toolbar--m-page-insets--xl--inset);
+  }
+
+  position: relative;
+  display: flex;
+  flex-wrap: var(--pf-c-masthead--FlexWrap);
+  align-items: center;
+  min-width: 0;
+  background-color: var(--pf-c-masthead--BackgroundColor);
+
+  @media screen and (min-width: $pf-global--breakpoint--md) {
+    --pf-c-masthead--FlexWrap: var(--pf-c-masthead--m-display-inline--FlexWrap);
+    --pf-c-masthead__main--Order: var(--pf-c-masthead--m-display-inline__main--Order);
+    --pf-c-masthead__main--FlexBasis: var(--pf-c-masthead--m-display-inline__main--FlexBasis);
+    --pf-c-masthead__main--BorderBottom: var(--pf-c-masthead--m-display-inline__main--BorderBottom);
+    --pf-c-masthead__main--PaddingRight: var(--pf-c-masthead--m-display-inline__main--PaddingRight);
+    --pf-c-masthead__main--PaddingLeft: var(--pf-c-masthead--m-display-inline__main--PaddingLeft);
+    --pf-c-masthead__content--PaddingRight: var(--pf-c-masthead--m-display-inline__content--PaddingRight);
+    --pf-c-masthead__content--PaddingLeft: var(--pf-c-masthead--m-display-inline__content--PaddingLeft);
+  }
+
+  > :first-child {
```

Comment:
I wonder if this is going to end up being fragile, like the table's first/last child selectors for insets. Seems like we could end up adding things that are hidden or shouldn't take on that spacing and run into issues.

Reply:
Great point, let me work on that a bit!